src/scripts/main.js

```diff
@@ -2,6 +2,30 @@
 
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+
+  const docBody = document.querySelector('body');
+
+  const notificationBox = document.createElement('div');
+  const notificationTitle = document.createElement('h2');
+  const notificationContect = document.createElement('p');
+
+  notificationBox.style.top = posTop + 'px';
+  notificationBox.style.right = posRight + 'px';
+
+  notificationTitle.textContent = title;
+  notificationTitle.classList.add('title');
+
+  notificationContect.textContent = description;
+
+  notificationBox.append(notificationTitle);
+  notificationBox.append(notificationContect);
+  notificationBox.classList.add('notification', type);
+
+  docBody.append(notificationBox);
+
+  setTimeout(() => {
+    notificationBox.style.display = 'none';
```

Comment:
Setting the CSS `display` property to `none` will only hide the notification element, not remove it from the DOM. Use `notificationBox.remove()` instead to remove the element from the DOM.